### ISSUE TITLE
Requirement already up to date when running pip install --upgrade tensorflow. Stuck at version 1.14.0

### ISSUE THREAD

**Issue by rolflocher** [tensorflow/tensorflow#41083]
<em>Please make sure that this is a build/installation issue. As per our [GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md), we only address code/doc bugs, performance issues, feature requests and build/installation issues on GitHub. tag:build_template</em>

**System information**
- OS Platform and Distribution: Rasbian GNU/Linux 10
- TensorFlow installed from: python3 -m pip
- TensorFlow version: 1.14.0 trying to get to 2.+
- Python version: 3.7.3
- Installed using virtualenv: pip
- CUDA/cuDNN version: Not sure, this is an off-the-shelf Raspberry Pi 4
- GPU model and memory: ^

**Describe the problem**

**Provide the exact sequence of commands / steps that you executed before running into the problem**
I tend to write scripts on my MacBook Pro and transfer them to my Raspberry Pi to run indefinitely. I set up my first LTSM network (so excited) and got it predicting and training. Brought it to the Pi, it threw an error while training complaining about the shape. After much googling I found a github issue where someone said this was fixed in tensorflow 2. Realized the pi was running tensorflow 1.14.0.

Long story short I've gone through all of the system requirements (upgrade pip, python is correct version, raspbian is correct version) and when I run `pip install --upgrade tensorflow` the console prints `Requirement already up to date: tensorflow....` and tensorflow remains at 1.14.0.

**Any other info / logs**
I figure this is a system requirement issue but it seems like the Raspberry Pi satisfies everything.

**Comment by rolflocher** [comment]
I've been troubleshooting this for a few hours now. I tried to manually download the pip wheels and for the linux wheel and both rpi wheels that I tried pip install throws "xxxx.whl is not a supported wheel on this platform"
Is it because I am using a Raspberry Pi 4 and not a Raspberry Pi 3?

**Comment by tensorflowbutler** [comment]
Hi There,

 We are checking to see if you still need help on this, as you are using an older version of tensorflow which is officially considered end of life . We recommend that you upgrade to the latest 2.x version and let us know if the issue still persists in newer versions. Please open a new issue for any help you need against 2.x, and we will get you the right help. 

 This issue will be closed automatically 7 days from now. If you still need help with this issue, please provide us with more information.

**Comment by KopfKrieg** [comment]
The issue is still relevant since there are no working Tensorflow 2.x builds for Python 3.7 (or newer) on the Raspberry Pi.

**Comment by KopfKrieg** [comment]
Again: Please reopen the issue, it's still not resolved.

I mean I would upgrade to TF2.x, but it's either not available or not working.

**Comment by mohantym** [comment]
Please go through this link for [reference](https://nitin9809.medium.com/getting-started-with-raspberry-pi-installation-guide-for-tensorflow-2-3-1-and-opencv-4-5-1-3a5bd0b6de2d) . Can you also try with these follow steps
1.upgrade your pip to latest version 
2.sudo pip install --upgrade tensorflow

if that does not work , uninstall then install latest version from above link.
